src/gwt/src/org/rstudio/studio/client/workbench/ui/PaneManager.java

```diff
@@ -403,8 +403,8 @@ else if (numDocs > 0 && window.getState() == WindowState.HIDE)
 
       eventBus.addHandler(UpdateTabPanelsEvent.TYPE, event ->
       {
-         left_.replaceWindows(panes_.get(0), panes_.get(1));
-         right_.replaceWindows(panes_.get(2), panes_.get(3));
+         left_.replaceWindows(panes_.get(0), panes_.get(1), DualWindowLayoutPanel.ReplaceMode.IF_DIFFERENT);
+         right_.replaceWindows(panes_.get(2), panes_.get(3), DualWindowLayoutPanel.ReplaceMode.IF_DIFFERENT);
```

Comment:
Looking into it, we don't seem to ever fire `UpdateTabPanelsEvent` so I think we should get rid of this handler and the event.

Reply:
Done!